corehq/apps/export/models/new.py

```diff
@@ -883,6 +886,14 @@ def _generate_schema_from_case_property_mapping(case_property_mapping, app_id, a
                 path=MAIN_TABLE,
                 last_occurrences={app_id: app_version},
             )
+
+            for system_prop in MAIN_CASE_TABLE_PROPERTIES[0]:
```

Comment:
Yeah I don't love it either. I actually went with `TOP_MAIN_FORM_TABLE_PROPERTIES`, and `BOTTOM_MAIN_FORM_TABLE_PROPERTIES` for the main form table, because I couldn't decide which I like better. Whatever I do, I should definitely standardize it. Another option, would be to put some sort of sentinel value in the list that means "dynamic properties go here", then we could do something like:

```
DYNAMIC_PROPERTIES = "this could be any sort of sentinel"
MAIN_FORM_TABLE = [
    [
        ...
    ],
    DYNAMIC_PROPERTIES,
    [
        ...
    ]
]


for property_group in MAIN_FORM_TABLE:
    if property_group == DYNAMIC_PROPERTIES:
        ...
    else:
        for system_prop in property_group:
            ...
```

Although I don't know if this is any better. I think I'll go with the TOP and BOTTOM approach unless you think the sentinel scheme is an improvement.